modular_zubbers/code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -0,0 +1,205 @@
+
+/datum/reagent/medicine/inaprovaline
+	name = "Inaprovaline"
+	description = "Stabilizes the breathing of patients. Good for those in critical condition."
+	reagent_state = LIQUID
+	ph = 8.5
+	color = "#5dc1f0"
+	overdose_threshold = 60
```

Comment:
You don't need to re-assign LIQUID to `reagent_state` for every reagent, this var is set on the parent, `/datum/reagent` already. (`/datum/reagent/medicine` just re-defines `taste_description`)

https://github.com/Bubberstation/Bubberstation/blob/22901d196fe86ef95292306c569d07a354608f7a/code/modules/reagents/chemistry/reagents.dm#L18-L32

Reply:
Side effect of copying code, I can make that change.

Reply:
That change has been made